### PR TITLE
LET-288 contact email & phone

### PR DESCRIPTION
**Pull request by agnessa**
https://vizzuality.atlassian.net/browse/LET-288?atlOrigin=eyJpIjoiZmJlMTY4OGJhM2M5NDEyOWFkOTcyNDUyOWI0ZmZlMDkiLCJwIjoiaiJ9

### CODE REVIEW

File: backend/app/models/account.rb [hunk 1]
```diff
@@ -22,6 +22,7 @@ class Account < ApplicationRecord
   validates :instagram, url: true
   validates :language, inclusion: {in: Language::TYPES}
   validates :picture, attached: true, content_type: /\Aimage\/.*\z/
+  validates :contact_email, presence: true
```
Comment (tsubik) [inline backend/app/models/account.rb:25]: Do we need email validation? I think devise is doing that with regex, but I'm not sure.

Comment (tsubik) [reply backend/app/models/account.rb:25]: `Devise.email_regexp`

Comment (agnessa) [reply backend/app/models/account.rb:25]: sure, added!

**Review by tsubik — APPROVED** [review]
:+1:

**Pull request merged**